plugins/contact-resources/src/index.ts

```diff
@@ -286,7 +286,10 @@ async function kickEmployee (doc: Person): Promise<void> {
     )
   }
 }
-async function openChannelURL (doc: Channel): Promise<void> {
+async function openChannelURL(doc: Channel): Promise<void> {
+  if (doc.value === undefined) {
+    return
```

Comment:
@haiodo How to fix the formatting CI check?

Reply:
`rush format` should be called locally, for entire project. and `rushx format` for individual ones.

Reply:
`rush format` is giving an error in a file unchanged by me. `rushx format` on the current file produces no changes